src/components/TokenGrid.tsx

```diff
@@ -23,10 +23,14 @@ const TokenGrid: React.FC<TokenGridProps> = () => {
   useEffect(() => {
     setWindowWidth(window.innerWidth);
     setWindowHeight(window.innerHeight);
-    window.addEventListener("resize", (e) => {
+    const resizeListener = (_e: any) => {
       setWindowHeight(window.innerHeight);
       setWindowWidth(window.innerWidth);
-    });
+    };
+    window.addEventListener("resize", resizeListener);
+    return () => {
+      window.removeEventListener("resize", resizeListener);
```

Comment:
fixes a react warning about leaked subscribers

Reply:
perf got that once too